build/filter_test.go

```diff
@@ -279,4 +281,635 @@ var _ = Describe("Test filters utilities:", func() {
 
 	})
 
+	Describe("Test  fileSystemFilter", func() {
+		It("", func() {
```

Comment:
missing "Context" intentionally? also empty It""
i think its good to have from readability perspective.

Reply:
Updated